.vale/styles/RedHat/Spelling.yml

```diff
@@ -49,7 +49,7 @@ filters:
   - "[dD]ev"
   - "[dD]ev[wW]orkspace"
   - "[dD]evfile"
-  - "[dD]evfiles"
```

Comment:
You removed devfiles

Did you do it on purpose?

@aireilly

Reply:
Yes. You don't need to add simple plurals in Spelling rule.

Reply:
OK, do you have a documentation you could share about this? I'm interested